### DISCUSSION TITLE
Is there a way to disable/inactive a component or project?

### DISCUSSION THREAD

**Discussion by hicksjacobp** [WeblateOrg/weblate#10551] [Q&A]
We have a Weblate instance which has a variety of projects and components over the years but are no longer active projects. We'd like to keep the translation history and reviewer activity around, but we don't care to actively manage these Weblate projects/components and we don't care about alerts for these.

I did a brief search through documentation and settings, but can't seem to find any kind of "disable" or "inactivation" setting for a project or component. Does such a thing exist? We'd like to reduce the noise from projects/components we don't care about and focus on the projects/components that are actively managed.

**Comment by nijel** [comment]
You can remove the project from watched ones or mute notifications for a project. But that's all per-user configuration.

**Reply by hicksjacobp** [reply]
If I would grab the JSON of a component via the API and also dump translation memory from Weblate, would that be a sufficient backup that could be reinstated later if necessary?

Would importing translation memory later be some kind of merge, or would it replace content from other components that would be created after dumping?

**Reply by nijel** [reply]
If this is about whole project,  using project backups is a better approach.
